build.gradle

```diff
@@ -149,6 +155,12 @@ subprojects {
             }
         }
     }
+    if (gradle.usingGeodeCompositeBuild) {
```

Comment:
Can we just bump the max heap regardless. If they intend to run examples they must have enough RAM to run geode so they must have enough RAM for us to always use more on the build.

Reply:
More accurately, if they are going to *build* geode, then they are already getting a max-heap bump from that Gradle build. But there is no guidance on how much memory is needed to run the examples, other than the low max-heap required to build them.
I think raising the value regardless of `-Dcomposite` addresses an issue we don't see.